Lesson3_Ex2/Program.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+
+namespace Lesson3_Ex2
+{
+    abstract class Figure
+    {
+        public abstract int AreaCalculator();
+        public abstract int PerimeterCalculator();
+    }
+    
+    class Program
+    {
+        static void Main()
+        {
+
```

Comment:
??? if there are any questions we could discuss it :)

Reply:
too many words in the task....